configs/config.protectli_pt201

```diff
@@ -1,4 +1,29 @@
 CONFIG_LOCALVERSION="4.12"
 CONFIG_VENDOR_PROTECTLI=y
+CONFIG_CBFS_SIZE=0x200000
+CONFIG_CONSOLE_CBMEM_BUFFER_SIZE=0x20000
+CONFIG_EDK2_BOOT_TIMEOUT=3
 CONFIG_BOARD_PROTECTLI_PT201=y
+CONFIG_EDK2_BOOTSPLASH_FILE="3rdparty/dasharo-blobs/dasharo/bootsplash.bmp"
```

Comment:
It needs protectli logo

Reply:
Changed it to protectli, but the problem is we don't have jsp on blobs repository yet, and the one in `vault/` fails to build (BMP compression error in edk2).